go/dpagg/select_partition.go

```diff
@@ -130,8 +130,17 @@ type PreAggSelectPartitionOptions struct {
 	// Optional.
 	PreThreshold int64
 	// MaxPartitionsContributed is the number of distinct partitions a single
-	// privacy unit can contribute to. Required.
+	// privacy unit can contribute to. For PreAggSelectPartition, MaxPartitionsContributed
```

Comment:
nit: "For PreAggSelectionPartition, setting MaxPartitionsContributed is functionally the same as setting MaxContributions, but are kept separate for consistency with other aggregations."

(also the same below)